server/registry/internal/storage/save.go

```diff
@@ -18,55 +18,67 @@ import (
 	"context"
 
 	"github.com/apigee/registry/server/registry/internal/storage/models"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 	"gorm.io/gorm/clause"
 )
 
-func (c *Client) SaveProject(ctx context.Context, v *models.Project) error {
+// SaveProject will return codes.NotFound if key not found
+func (c *Client) SaveProject(ctx context.Context, v *models.Project, fieldMask []string) error {
```

Comment:
Is there a benefit to accepting `[]string` here instead of `*fieldmaskpb.FieldMask`? The only difference this seems to make is whether we call `GetPaths()` in the action handler or in the `saveWithMask` method, so I lean toward keeping the more specialized type with all of its documentation about what a field mask is.

Reply:
In retrospect, I guess there's no point in avoiding the dependency in this package... we rely on the protobuf stuff everything. I'll change it.

Reply:
Done